sdk/nodejs/dynamic/index.ts

```diff
@@ -21,11 +21,11 @@ import * as serializeClosure from "../runtime/closure/serializeClosure";
 /**
  * CheckResult represents the results of a call to `ResourceProvider.check`.
  */
-export interface CheckResult {
+export interface CheckResult<Inputs> {
```

Comment:
Don't all of these need to default to any as well? `<Inputs = any>`, otherwise existing untyped declarations of them will change to `unknown` instead of `any`.

Reply:
You're right, if anybody uses those types directly, so that has to be fixed, oversigt on my part.

Perhaps I can add tests anyway, to ensure we don't break anything.

Reply:
I've added a test case that secures the old usage is still valid, even with explicit types used.

And verifies proper type-safety when passing a generic type the the interface and using the inferred types.